src/i18n/man.json

```diff
@@ -0,0 +1,82 @@
+{
+  "locale": "man",
+  "lang_name": "简体中文",
+  "lang_name_in_eng": "Mandarin",
+  "language": "语言",
+  "welcome": "欢迎",
+  "welcome_blurb": "双城相助 （Twin Cities Mutual Aid）是关于明尼阿波利斯和圣保咯两座城人民市相助地点的最新目录。",
+  "lang_select": "选择语言",
+  "page_title": "双城相助地图",
+  "show_list_button": "显示地点单子",
+  "hide_list_button": "关闭地点单子",
+  "sort_by": "用类目分别",
+  "sort_by_urgent_need": "紧急请求",
+  "sort_by_name": "",
```

Comment:
are we just missing this translation?

Reply:
Looks addressed in comment, missed that. There seemed to be 4, though, including "time_future"